astropy/timeseries/bls/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
I would suggest to structure these on namespace below? They'll still be imported in `timeseries/__init__.py`, yet the code has a bit more structure.

Reply:
Fixed